athlinks-schema.ttl

```diff
@@ -145,10 +161,48 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:Duration .
 
 athlinks:seriesType
   a owl:DataTypeProperty;
-  rdfs:range[ a xsd:string ;
-		owl:oneOf("Master", "Social", "Points", "Series")
-	     ] .
+  rdfs:range [ a xsd:string ;
+								owl:oneOf("Master", "Social", "Points", "Series")
+	     				] .
+
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
```

Comment:
Those seem to be part of the race in most systems that we have looked at BUT if we create something that holds the registration separate from the race then I think I see what you are saying.